about.html

```diff
@@ -0,0 +1,138 @@
+            <div id="xvalue" class="close-value">&#10005;
+            </div>
+            <nav id="menunav" class="menu-nav" >
+                    <ul class="menu-content">
+                        <a class="menu-content-list"  href="about.html">About</a>
+                        <a class="menu-content-list"  href="index.html#programid">Program</a>
+                        <a class="menu-content-list"  href="#aboutcardid">join</a>
+                        <a class="menu-content-list" href="#partenerid">Sponser</a>
+                        <a class="menu-content-list" href="#contactcard">News</a>
+                        <a class="menu-content-list" href="#contactcard">CC Campaign</a>
+                    </ul>
+            </nav>
+        </div>
+    <section id="aboutid" class="home">
+                <div>
+                    <h2>"Hello! Sharing word"</h2>
+                    <h1>ICT Expo Ethiopia 2023</h1>
+                </div>
+                <div>
+                    <p>  ICT Expo Ethiopia brings the latest technology innovations to benefit organizations operating or 
+                        planning to engage in the expanding Ethiopian market. The Expo serves as a catalyst 
+                        for collaboration and coopetition between foreign and local technology companies   </p>
+                </div>
+                <div>
+                    <p> Please contact us per Email for any further question about  ICT Expo Ethiopia 2023 !</p>
+                    <a href="">damenam@gmail.com</a>
+                </div>
+    </section>
+  
+    <section >
+        <h2> ICT Expo Ethiopia 2023  </h2>
+        <p>Innovate Ethiopia   May 14 – May 18, 2020 </p>
+            <div class="expo-eth">
+                <img class="logo-2015" src="./image/ICT-Logo-2015.jpg" alt="logoExpo">
+            </div>
+    </section>
+   
+    <section id="newsid">
+        <h2 class="see-past">see the past ICT Expo Ethiopia </h2>
+        <div class="line"></div>
+        <p class=""> ICT Expo Ethiopia is where established technology service vendors integrators, and users of ICT technology for business or public </p>
+                <div class="last-exp">
+                    <div class="first-exp">
+                        <div>
+                            <img src="./image/Past-expo1.jpg" alt="" srcset="">
+                        </div>
+                        <div class="center">
+                            <h2>2022</h2>
+                            <p>ICT Expo Ethiopia 2023 Addis Abeba</p>
+                        </div>
+                    </div>
+                    <div class="first-exp">
+                        <div>
+                            <img src="./image/Past-Expo2.jpg" alt="" srcset="">
+                        </div>
+                        <div  class="center">
+                            <h2>2021</h2>
+                            <p>ICT Expo Ethiopia 2023 Mekele</p>
+                        </div>
+                       
```

Comment:
- [ ] Please change the overlay color to orange as it is on the design given. Look at the comparison 👇 
Your design
![image](https://user-images.githubusercontent.com/90222110/212418859-aebb32bf-a107-4368-bda4-751941edd093.png)
Given design
![image](https://user-images.githubusercontent.com/90222110/212418908-768c842d-92d5-4614-843e-985eb8b03813.png)